library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -215,6 +215,20 @@ button[class="disabled"]:hover,
 	text-decoration: none;
 }
 
+input[type="checkbox"].formButtonDisabled{
+	@include border-radius(3px);
+	&:checked{
+		background-image: url("images/check.png") !important;
```

Comment:
Yes, I agree, but this is the only way I could get it to override another !important. I'm open to suggestions.